install/installing-with-kata-manager.md

```diff
@@ -0,0 +1,24 @@
+# Installing with kata-manager
+
+* [Introduction](#Introduction)
+* [Install the Kata packages](#install-the-kata-packages)
+* [Install Docker](#install-docker)
+
+## Introduction
+If your distribution is supported, you can use [`kata-manager`](https://github.com/kata-containers/tests/blob/master/cmd/kata-manager)
+to avoid copying or typing installation instructions by hand.
+
+## Install the Kata packages
+```
+bash -c "$(curl -fsSL \
+https://raw.githubusercontent.com/kata-containers/tests/master/cmd/kata-manager/kata-manager.sh) \
+install-packages"
+```
+
+## Install Docker
+```
+bash -c "$(curl -fsSL \
+https://raw.githubusercontent.com/kata-containers/tests/master/cmd/kata-manager/kata-manager.sh) \
+install-docker-system"
```

Comment:
Note that `install-docker-system` implies `install-packages`. To get a fully working system, you *only* need to run  `install-docker-system` so you could drop the `install-packages` section, or move it *after* the "Install Docker" section to say that if your chosen container manager is not Docker, you can install the Kata packages but need to also install k8s, etc.